tests/Oryx.Integration.Tests/LocalDockerTests/NodeEndToEndTests.cs

```diff
@@ -607,6 +607,46 @@ public async Task Node_CreateReactAppSample()
                 });
         }
 
+        [Theory]
+        // TODO - renable it with 851972, removing the inline data
+        //[MemberData(nameof(TestValueGenerator.GetNodeVersions), MemberType = typeof(TestValueGenerator))]
+        [InlineData("10.10")]
+        [InlineData("10.14")]
+        public async Task Node_CreateReactAppSample_zippedNodeModules(string nodeVersion)
+        {
+            // Arrange
+            var appName = "create-react-app-sample";
+            var hostDir = Path.Combine(_hostSamplesDir, "nodejs", appName);
+            var volume = DockerVolume.Create(hostDir);
+            var appDir = volume.ContainerDir;
+            var portMapping = $"{HostPort}:{ContainerPort}";
+            var runAppScript = new ShellScriptBuilder()
+                .AddCommand($"cd {appDir}")
```

Comment:
Probably just create an empty node_modules under this directory so that we know that it is not used.